include/openspace/util/touch.h

```diff
@@ -56,6 +56,7 @@ class TouchInputHolder {
     // tryAddInput:
     // Succeeds upon a different input than last.
     // Fails upon a too similar input as last.
+    // Updates time for the last input if same position.
```

Comment:
I would prefer to see more comments like these in this file for the TouchInput and TouchInputHolder classes. Not every method needs an explanation, but it would help for some like holdsInput().

Reply:
This was a good idea, I added some explanation to the TouchInput and the TouchInputHolder as well to differentiate them.